tests/conftest.py

```diff
@@ -116,6 +116,10 @@ def test_with_any_microvm(test_microvm_any):
 session root and temporary directories.
```

Comment:
@marcov, this is commit is nice functionality. I encourage you to make a different PR containing just this commit since it s totally decoupled from the initrd support and it has higher chances of getting merged faster.

Reply:
Sure!